srcpkgs/libbitcoin-client/template

```diff
@@ -0,0 +1,56 @@
+# Template file for 'libbitcoin-client'
+maintainer="Andy Weidenbaum <archbaum@gmail.com>"
+pkgname="libbitcoin-client"
+version=3.5.0
+revision=1
+short_desc="Bitcoin Client Query Library"
+makedepends+=" boost-devel"
```

Comment:
use just makedepends="all the depenencies in a single string
 but cut at 80 columns and in the next line add a whitespace
 caracther like it is being done now"